PictureApp/PictureService.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PictureService.swift
+//  PictureApp
+//
+//  Created by Irina Gorbachenko on 01.12.2021.
+//
+
+import UIKit
+import NetworkLayer
+
+
+public struct PictureService {
+
+    let session: HTTPClient
+
+   public init(with session: HTTPClient) {
+        self.session = session
+    }
+
+    func fetch(completion: @escaping ([Post], Int) -> (),order : Order ,selectedCategory:[String],currentPage:Int) {
+
+        var urlComponents = URLComponents(string: "https://pixabay.com/api")!
+        urlComponents.queryItems = [
```

Comment:
Replace it with for (for study proposes)

Reply:
Done